.github/workflows/jacoco_check.yml

```diff
@@ -45,11 +45,11 @@ jobs:
         with:
           java-version: "adopt@1.8"
       - name: Build and run tests
-        run: sbt ++${{matrix.scala}} jacoco
+        run: sbt ++${{matrix.scala}} "agent/jacoco"
```

Comment:
This line produces only build for 2.13. So there are missing XML report values.

Reply:
@miroslavpojer 
Do you happen to know then how to trigger it correctly only for agent module?

Reply:
changed to 
```sh 
sbt "++${{matrix.scala}}; project agent; jacoco"
```